### ISSUE TITLE
Playback Stutters

### ISSUE THREAD

**Issue by ProjectMoon** [monofox/harbour-audioaddict#2]
Most noticeable on high quality, but also on medium (didn't test low).

This was with Digitally Imported channels.

There's no noticeable CPU usage according to Crest.

Device: Sony Xperia 10 Plus

**Comment by monofox** [comment]
Hi @ProjectMoon ,
thanks for your report.
Indeed i face this issue on Xperia 10 as well and experienced it with the former SkyFm-App too.

I didn't found the root cause yet (either e.g. QML Audioplayer is not fast enough to handle the network stream on Sailfish or if e.g. the underlaying component of the network stream is not caching enough).

**Comment by ProjectMoon** [comment]
Is caching of the stream handled automatically? Maybe it needs some tweaking?

**Comment by monofox** [comment]
Yes @ProjectMoon. Caching is right now automatically handled by Qt (via https://doc.qt.io/qt-5/qml-qtmultimedia-audio.html). I didn't found any way to tweak the cache from QML side. 
Original i thought, that the issue is due to the high amount of parallel requests for the stations pictures and implemented a caching algorithm. Now the channels are pretty fast loaded, but it still stucks. 
Therefore i guess, i must somehow find a way through C routines to tweak the cache or to implement custom network provider for audio player.
